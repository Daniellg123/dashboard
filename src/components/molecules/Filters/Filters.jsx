import { Button, Typography } from "@mui/material";
import { Funnel } from "phosphor-react";

function Filters() {
    return(
        <Button variant="outlined" sx={{ borderColor: "#C4C8CF",}}>
            <Funnel size={17}/>
            <Typography variant="body1">Filters</Typography>
        </Button>
    );
};

export default Filters;