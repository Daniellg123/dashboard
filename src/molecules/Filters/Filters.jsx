import { Button } from "@mui/material";
import { Funnel } from "phosphor-react";

function Filters() {
    return(
        <Button variant="outlined">
            <Funnel size={17}/>
            Filters
        </Button>
    );
};

export default Filters;