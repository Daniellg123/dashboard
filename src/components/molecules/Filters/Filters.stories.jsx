import Filters from "./Filters";

export default {
    title: 'Molecules/Filters',
    component: Filters,
};

export const filter = () => <Filters />