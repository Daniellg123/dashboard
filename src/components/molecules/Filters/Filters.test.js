import renderer from 'react-test-renderer';
import Filters from './Filters';

it('renders correctly', () => {
    const render = renderer.create(
        <Filters />
    ).toJSON();
    expect(render).toMatchSnapshot();
});