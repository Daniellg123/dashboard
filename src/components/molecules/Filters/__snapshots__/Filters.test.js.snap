// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<button
  className="MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButtonBase-root  css-17sr608-MuiButtonBase-root-MuiButton-root"
  disabled={false}
  onBlur={[Function]}
  onContextMenu={[Function]}
  onDragLeave={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex={0}
  type="button"
>
  <svg
    fill="currentColor"
    height={17}
    viewBox="0 0 256 256"
    width={17}
    xmlns="http://www.w3.org/2000/svg"
  >
    <rect
      fill="none"
      height="256"
      width="256"
    />
    <path
      d="M42.1,48H213.9a8,8,0,0,1,5.9,13.4l-65.7,72.3a7.8,7.8,0,0,0-2.1,5.4v56.6a7.9,7.9,0,0,1-3.6,6.7l-32,21.3a8,8,0,0,1-12.4-6.6v-78a7.8,7.8,0,0,0-2.1-5.4L36.2,61.4A8,8,0,0,1,42.1,48Z"
      fill="none"
      stroke="currentColor"
      strokeLinecap="round"
      strokeLinejoin="round"
      strokeWidth="16"
    />
  </svg>
  <p
    className="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
  >
    Filters
  </p>
</button>
`;
